### DISCUSSION TITLE
Can experimental CSS properties be used?

### DISCUSSION THREAD

**Discussion by uknmr** [tailwindlabs/tailwindcss#14139] [Help] [answered]
Is there a way to use "text-spacing-trim" in tailwindcss? No matter how I code it up, tailwind ignores it.

We want to fix with the Yu Gothic bug in WIndows.

https://caniuse.com/mdn-css_properties_text-spacing-trim

**Comment by wongjn** [comment, marked as answer]
You can use it in an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) like `[text-spacing-trim:space-all]` or [write a Tailwind plugin](https://tailwindcss.com/docs/plugins#adding-utilities) that would add some utility classes that apply the values.

**Reply by uknmr** [reply]
Thank you for your help!
We used `addBase` to solve the problem.

```js
addBase({
  body: {
    textSpacingTrim: 'space-all',
  },
})
```
